### PR TITLE
TTS: Gas Mask muffling, Hailer Mask voice effects, support for more filters that use samplerate, voice effects for lizards, ethereals, and xenomorphs.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

https://github.com/tgstation/tgstation/assets/4081722/7c8df2e2-8795-4048-b11e-e831f749a097

https://github.com/tgstation/tgstation/assets/4081722/d0bec7fd-25b1-4bdd-98a6-0ebfea749551

https://github.com/tgstation/tgstation/assets/4081722/31c0259a-2bd4-4575-a758-a2044ccf17b0

## Why It's Good For The Game

Improves TTS audio immersion.

## Changelog
:cl:
qol: Gas masks now muffle your voice with TTS.
qol: Security Hailer masks now disguise your voice to protect your right to brutalize greytiders.
qol: Lizards, Ethereals, and Xenomorphs now have a vocal effect.
qol: Security Records now show someone's voice name.
/:cl:

### CODE REVIEW

**Comment by Vincent983** [comment]
Does the sec-hailer apply it's voice filters while adjusted to not be on your face?

**Comment by optimumtact** [comment]
I can already hear the john madden aeiou spam the round this gets enabled.

File: tools/tts/tts-api/tts-api.py [hunk 1]
```diff
@@ -102,4 +130,12 @@ def pitch_available():
...
 		os.putenv('LD_LIBRARY_PATH', os.getenv('TTS_LD_LIBRARY_PATH'))
 	from waitress import serve
+	#response = requests.get(f"http://127.0.0.1:5003/tts-voices")
+	#list_of_voices = json.loads(response.content)
+	#for voice in list_of_voices:
+	#	for pitch in [-12, 0, 12]:
+	#		dummy_request = requests.get(f"http://127.0.0.1:5003/generate-tts-blips", json={ 'text': "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789", 'voice': voice, 'pitch': str(pitch) })
+	#		if dummy_request.status_code != 200:
+	#			print(dummy_request.status_code)
+	#			print("FAILED TO DO PITCH " + str(pitch) + " VOICE " + str(voice))
```
Comment (Watermelon914) [inline tools/tts/tts-api/tts-api.py:140]: just remove this if we're not going to keep it in

**Comment by Serxule** [comment]
lizard and xeno dont really sound good, at all, it just sounds like a generic free voice changer, but the rest is good

**Comment by Dorsisdwarf** [comment]
why are the low quality hi pitch voicechanger filters from a previous cancelled PR being packaged with the quite nice new TTS filters/muffling effects?

**Comment by TiviPlus** [comment]
Can you make the special_filters filters into defines please

**Comment by TiviPlus** [comment]
sidenote istype(src) is stinky

**Comment by Iamgoofball** [comment]
@TiviPlus filters are now defines

File: tools/tts/tts-api/tts-api.py [hunk 2]
```diff
@@ -93,7 +121,7 @@ def pitch_available():
 	if authorization_token != request.headers.get("Authorization", ""):
 		abort(401)
 
-	response = requests.get(f"http://tts-container:5003/pitch-available")
+	response = requests.get(f"http://127.0.0.1:5003/pitch-available")
```
Comment (ninjanomnom) [inline tools/tts/tts-api/tts-api.py:124]: This should probably be config

**Pull request merged**